src/lib/es2015.iterable.d.ts

```diff
@@ -5,7 +5,7 @@ interface SymbolConstructor {
      * A method that returns the default iterator for an object. Called by the semantics of the
      * for-of statement.
      */
-    readonly iterator: symbol;
+    readonly iterator: unique symbol;

```

Comment:
Fun fact! I didn't actually need to make any of these lib changes to make the PR work, what with the compat code in place! But I've updated them anyway, so they serve as examples of what `unique symbol`s really are to anyone reading the definitions.

Reply:
Hm - and that won't conflict with a separate "polyfill" definition right? In other words, if someone separately declares it as

```ts
interface SymbolConstructor {
    readonly iterator: symbol;
}
```

and these merge, they won't end up conflicting, right?

Reply:
Nope - and there's already a test to that effect on the diff.